redis/asyncio/connection.py

```diff
@@ -764,10 +764,10 @@ async def send_packed_command(
                 f"Error {err_no} while writing to socket. {errmsg}."
             ) from e
         except BaseException:
-            # BaseExceptions can be raised when a send*() is not finished, e.g. due
+            # BaseExceptions can be raised when a send() is not finished, e.g. due
```

Comment:
technically `sendall`, no? :)

Reply:
Yes, but I'm referring to the sort of classic POSIX socket API.    I guess I could just say "socket send" to keep it ambiguous enough and avoid confusion.

Reply:
there, I added an update.  maybe it is better?